tests/data/small_sky_to_o1source/catalog_info.json

```diff
@@ -6,7 +6,7 @@
     "primary_column": "id",
     "primary_column_association": "object_id",
     "join_catalog": "small_sky_order1_source",
-    "join_column": "source_id",
+    "join_column": "object_id",
```

Comment:
HATS change, or an additional refactoring during renaming?

Reply:
Hm. Not sure. The old value looks wrong, and the new value looks right, since this is the column to use for merging to the joining table.